### ISSUE TITLE
static windows binary release

### ISSUE THREAD

**Issue by kzhsw** [eclipse/mosquitto#2957]
Currently the [win32](https://mosquitto.org/files/binary/win32/) and [win64](https://mosquitto.org/files/binary/win64/) binary releases, after version `2.0.9a`, are packed with a `VC_redist.x64.exe`, making the release 10x larger then versions before that.
Linking the release with static flag, `/MT` for msvc, can reduce the overall size of binary, making it easier to download and deploy.

**Comment by ralight** [comment]
I don't believe it's a good idea to mix `/MD` and `/MT` binaries, so including the redistributable is the safest option. Yes the installer is bigger, but even on a slow connection it's not a lot
